src/IO/H5/VolumeData.cpp

```diff
@@ -808,6 +843,14 @@ std::optional<std::vector<char>> VolumeData::get_domain(
 
 std::optional<std::vector<char>> VolumeData::get_functions_of_time(
     const size_t observation_id) const {
```

Comment:
Change to `std::optional<size_t>`: if an observation_id is given, get the evaluated FoT stored in that observation only (backward-compatible and less data to load than before if all you need is spatial interpolation). If no observation_id is given, get the global FoT (only needed for time interpolation). Could make the case where no observation_id is given also backward-compatible by choosing the last observation ID in the file.

- `src/Domain/Creators/TimeDependentOptions/FromVolumeFile.cpp` needs to be updated to retrieve the FoT from the observation at the given `time` (not always get the last one).